lib/streamlit/elements/widgets/button.py

```diff
@@ -221,19 +221,31 @@ def button(
               of the parent container.
 
         shortcut : str or None
-            An optional keyboard shortcut that triggers the button. Provide a
-            single alphanumeric key (e.g. ``"K"``, ``"4"``), a function key
-            (e.g. ``"F11"``), or a supported special key (e.g. ``"Enter"``,
-            ``"Esc"``), optionally combined with modifiers.
+            An optional keyboard shortcut that triggers the button. This can be
+            one of the following strings:
 
-            Examples: ``"Ctrl+K"``, ``"Cmd+Shift+O"``, ``"Mod+Enter"``.
+            - A single alphanumeric key like ``"K"`` or ``"4"``.
+            - A function key like ``"F11"``.
+            - A special key like ``"Enter"``, ``"Esc"``, or ``"Tab"``.
+            - Any of the above combined with modifiers. For example, you can use
+              ``"Ctrl+K"`` or ``"Cmd+Shift+O"``.
 
-            .. note::
-                The keys ``"C"`` and ``"R"`` are reserved and cannot be used,
-                even with modifiers. ``"Ctrl"``, ``"Cmd"``, and ``"Mod"`` are
-                platform-dependent: they map to ``"Command"`` (⌘) on macOS and
-                ``"Control"`` on Windows/Linux. Punctuation keys (e.g. ``"."``,
-                ``","``) are not currently supported.
+            .. important::
+                The keys ``"C"`` and ``"R"`` are reserved and can't be used,
+                even with modifiers. Punctuation keys like ``"."`` and ``","``
+                aren't currently supported.
+
+            The following special keys are supported: Backspace, Delete, Down,
+            End, Enter, Esc, Home, Left, PageDown, PageUp, Right, Space, Tab,
+            and Up.
+
+            The following modifiers are supported: Alt, Ctrl, Cmd, Meta, Mod,
+            Option, Shift.
+
+            - Ctrl, Cmd, Meta, and Mod are interchangeable and will display to
+              the user to match their platform.
```

Comment:
Hello, I'm sorry to intervene here directly but this looks wrong 👀:
The Ctrl & Cmd keys are different keys on Mac computers, I don't understand why they would be considered the same way..
Or are you saying that Ctrl cannot be mapped at all on Macs and it'll always use Cmd instead?

Reply:
@bew while there is a ctrl key on Mac, it's not supposed to be used for custom keyboard shortcuts. Cmd on Mac is the primary modifier key, and it's common that `ctrl` gets mapped to `ctrl` on Windows/Linux and `cmd` on Mac.